internal/config/config.go

```diff
@@ -19,26 +20,21 @@ var (
 	MinersWhiteList = ""
 )
 
-// TODO: Revamp the whole config, using a Map and having multiple global functions to get the config is not nice
-func init() {
-	switch strings.ToLower(os.Getenv("LOG_LEVEL")) {
-	case "debug":
-		logrus.SetLevel(logrus.DebugLevel)
-	case "info":
-		logrus.SetLevel(logrus.InfoLevel)
-	case "warn":
-		logrus.SetLevel(logrus.WarnLevel)
-	case "error":
-		logrus.SetLevel(logrus.ErrorLevel)
-	default:
-		logrus.SetLevel(logrus.InfoLevel)
-	}
-}
+const defaultDataDir = "/home/masa"
+const defaultListenAddress = ":8080"
+
+// TODO: Revamp this whole thing, a map[string]any is not really maintainable
+type JobConfiguration map[string]any
 
-func ReadConfig() types.JobConfiguration {
+func ReadConfig() JobConfiguration {
 	// The jobs will then unmarshal from this configuration to the specific configuration
 	// that is needed for the job
-	jc := types.JobConfiguration{}
+	jc := JobConfiguration{}
+
+	logLevel := os.Getenv("LOG_LEVEL")
```

Comment:
does it make sense to bring this down into the general handling of `.env` file loading, maybe at the beginning of it?